Documentation/Profiles/Profiles.md

```diff
@@ -9,7 +9,8 @@ For example, the Input system's behavior is governed by its input profile. An ex
 of such a profile is [here](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityInputSystemProfile.asset). In order to edit this, it's usually recommended
 to use the in-editor inspector:
 
-![MRTK](../Images/Profiles/input_profile.png)
+<img src="../../Documentation/Images/Profiles/input_profile.png" width="650px" style="display:block;">
+<sup>Profile Inspector</sup>
 
 > **Note:** While it is intended that profiles can be swapped out at runtime, this currently
```

Comment:
Sorry, also same here and all places. This is our goal but not exactly accurate atm

Reply:
Updated